.github/dependabot.yml

```diff
@@ -73,6 +73,11 @@ updates:
     schedule:
       interval: "daily"
 
+  - package-ecosystem: "gradle"
+    directory: "implementations/java-json-schema-validator"
```

Comment:
The directory here is named differently than the real one, so this will fail to run I think -- let's actually keep this as is and rename the directory.